Sources/Parsing/Parser.swift

```diff
@@ -69,7 +134,62 @@ extension Parser {
     }.parse(input[...])
   }
 
-  /// Attempts to parse a nebulous collection of data into something more well-structured.
+  /// Parse a `String` into an output using a UTF-8 parser. This method is more ergnomic to use
+  /// than ``parse(_:)-76tcw`` because it accepts a plain string rather than a collection of
+  /// UTF-8 code units, and the input does not need to be `inout`.
+  ///
+  /// Rather than having to create a mutable UTF-8 value and feed it to the ``parse(_:)-76tcw``
+  /// method like this:
+  ///
+  /// ```swift
+  /// var input = "123,true"[...].utf8
+  /// let output = try Parse {
+  ///   Int.parser()
+  ///   ",".utf8
+  ///   Bool.parser()
+  /// }
+  /// .parse(&input) // (123, true)
+  /// ```
+  ///
+  /// You can just feed a plain `String` input directly:
+  ///
+  /// ```swift
+  /// let output = try Parse {
+  ///   Int.parser()
+  ///   ",".utf8
+  ///   Bool.parser()
+  /// }
+  /// .parse("123,true") // (123, true)
+  /// ```
+  ///
+  /// This method will fail if the parser does not consume the entirety of the input.
+  /// For example:
+  ///
+  /// ```swift
+  /// let output = try Parse {
+  ///  Int.parser()
+  ///  ",".utf8
+  ///  Bool.parser()
```

Comment:
```suggestion
  ///   Int.parser()
  ///   ",".utf8
  ///   Bool.parser()
```

Reply:
Should this be in a `> Tip:` like above?

Reply:
I thought we were leaving this part as-is and just making the whitespace thing a tip. You think both should be tips?